src/storage/kv/stats.rs

```diff
@@ -118,6 +109,7 @@ impl Statistics {
         self.data.add(&other.data);
     }
 
+    /// Deprecated
```

Comment:
Consider using [the deprecated decl](https://doc.rust-lang.org/edition-guide/rust-2018/the-compiler/an-attribute-for-deprecation.html)?

Reply:
Then CI will fail